polaris.shopify.com/pages/examples/data-table-with-fixed-first-columns.tsx

```diff
@@ -276,6 +277,8 @@ function DataTableWithFixedFirstColumnsExample() {
     ],
   ];
   const [sortedRows, setSortedRows] = useState(rows);
+  const showFixedColumns = useMedia('screen and (max-width: 850px)');
```

Comment:
I noticed this will not work when editing in codesandbox because of the `useMedia` hook. Is this something we should be worried about?

Reply:
Forgot to do it before merging 😅. I'll follow up with a another PR to fix the example for codesandbox.